tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -1,17 +1,19 @@
 package com.epam.pipeline.tesadapter.controller;
 
+import com.epam.pipeline.tesadapter.entity.TesCancelTaskResponse;
+import com.epam.pipeline.tesadapter.entity.TesCreateTaskResponse;
+import com.epam.pipeline.tesadapter.entity.TesTask;
 import com.epam.pipeline.tesadapter.service.TesTaskService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please, do not use "star" imports. You can configure it for Idea using `Class count to use import with '*'` option.

Reply:
fixed